### PR TITLE
T622: Ultraconnected compact spaces have a focal point

### PR DESCRIPTION
**Pull request by prabau**
New T622: Ultraconnected + compact + not empty ==> has a focal point.

That seemed simple enough to not warrant a mathse post.

Will provide a simple derivation that S100 is not compact (upcoming in #948).

Also will derive that S118 (integer broom) has a focal point.

### CODE REVIEW

**Review by david20000813 — APPROVED** [review]

**Comment by prabau** [comment]
@danflapjax Can you also review and merge?

**Comment by david20000813** [comment]
@prabau I realized. I did the review sort of as a test to see if I can do reviews yet, but unfortunately not.

**Comment by prabau** [comment]
> @prabau I realized. I did the review sort of as a test to see if I can do reviews yet, but unfortunately not.

Well, you can do a review and add comments and approve if you think it's ready to go.  That is very valuable information.
It is just that one also needs an additional (pro-forma) review from someone with write access.

**Review by danflapjax — APPROVED** [review]

**Pull request merged**
